### PR TITLE
Control Objective - Alarm on VPC Route Table public routes

### LINKED ISSUE #300: Control Objective - Alarm on VPC Route Table public routes
Create a terraform example where Turbot alarms if a VPC route table has public routes

https://github.com/turbot/tdk/pull/290

### PR DESCRIPTION
**Pull request by choxie**
closes #300

### CODE REVIEW

**Comment by omerosaienni** [comment]
No issue linked

**Review by omerosaienni — CHANGES_REQUESTED** [review]
No linked PR

**Comment by choxie** [comment]
closes https://github.com/turbot/tdk/issues/300

File: calculated_policies/aws_vpc_public_routes/README.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
+# AWS EC2 - Restrict Instance images to trusted AWS accounts AMIs
```
Comment (omerosaienni) [inline calculated_policies/aws_vpc_public_routes/README.md:1]: Documentation for this is out of sync, please review.

File: calculated_policies/aws_vpc_public_routes/main.tf [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  routeTable{
+    Routes{
+      DestinationCidrBlock
+      GatewayId
+    }
+  }
+}
+  EOT
+  template      = <<EOT
+{%- set cidrMatches = false -%}
```
Comment (omerosaienni) [inline calculated_policies/aws_vpc_public_routes/main.tf:23]: @choxie 

I have a pretty way on how to structure this code. If you request a meeting with you I can train you up.

**Review by omerosaienni — CHANGES_REQUESTED** [review]

File: calculated_policies/aws_vpc_public_routes/README.md (hunk 1 above)
Comment (choxie) [reply calculated_policies/aws_vpc_public_routes/README.md]: fixed in newest commit

**Pull request closed without merging**